CoreOpenGLDisplay/src/au/gov/asd/tac/constellation/visual/opengl/utilities/glyphs/GlyphManagerFX.java

```diff
@@ -57,9 +59,9 @@
  *
  * @author sirius
  */
-public class GlyphManager {
+public class GlyphManagerFX implements GlyphManager {
```

Comment:
Sooo... we can just get rid of this now.... right?

Reply:
After speaking with @algol60 it sounds like we should keep this around for a few months as a fallback.